### ISSUE TITLE
Notice in ./vendor/phpmyadmin/sql-parser/src/Parser.php#483

### ISSUE THREAD

**Issue by liviuconcioiu** [phpmyadmin/sql-parser#413]
Steps to reproduce:

1. Go to [https://demo.phpmyadmin.net/QA_5_2/index.php?route=/database/sql&db=sakila](https://demo.phpmyadmin.net/QA_5_2/index.php?route=/database/sql&db=sakila)
2. Paste `ANALYZE` - only with this word it can be reproduced.
3. Click on a database from the left side

Depending on settings, is either a warning or a notice. It happens on both QA_5_2 and master.

https://user-images.githubusercontent.com/25424343/213540973-04415631-9da4-43e6-b21d-072507dd723d.mp4

```
Notice in ./vendor/phpmyadmin/sql-parser/src/Parser.php#483
Trying to get property 'keyword' of non-object

Backtrace

Parser.php#377: PhpMyAdmin\SqlParser\Parser->parse()
Linter.php#126: PhpMyAdmin\SqlParser\Parser->__construct()
LintController.php#59: PhpMyAdmin\Linter::lint(string 'ANALYZE')
Routing.php#191: PhpMyAdmin\Controllers\LintController->__invoke(
,
array,
)
index.php#43: PhpMyAdmin\Routing::callControllerForRoute(
,
string '/lint',
,
,
)
```

```
Warning in ./vendor/phpmyadmin/sql-parser/src/Parser.php#483
Attempt to read property "keyword" on null

Backtrace

    Parser.php#377: PhpMyAdmin\SqlParser\Parser->parse()
    Linter.php#125: PhpMyAdmin\SqlParser\Parser->__construct()
    LintController.php#59: PhpMyAdmin\Linter::lint(string 'ANALYZE')
    Routing.php#167: PhpMyAdmin\Controllers\LintController->__invoke(
    ,
    array,
    )
    Common.php#325: PhpMyAdmin\Routing::callControllerForRoute(
    ,
    ,
    ,
    )
    index.php#32: PhpMyAdmin\Common::run()
```

**Comment by williamdes** [comment]
Do you think @liviuconcioiu we should make a parser release or wait for phpMyAdmin to be released first and then do it as we had planned?

**Comment by liviuconcioiu** [comment]
> Do you think @liviuconcioiu we should make a parser release or wait for phpMyAdmin to be released first and then do it as we had planned?

Depends how many people will use `ANALYZE`. I think a minor release should be done, before 5.2.1 is released.

**Comment by williamdes** [comment]
> > Do you think @liviuconcioiu we should make a parser release or wait for phpMyAdmin to be released first and then do it as we had planned?
> 
> Depends how many people will use `ANALYZE`. I think a minor release should be done, before 5.2.1 is released.

Yeah we could do a minor with just this fix into it 🤔 
Or release all the changes, most of the risk is released in the 5.6.0 version of the parser

**Comment by williamdes** [comment]
Released as [5.7.0](https://github.com/phpmyadmin/sql-parser/releases/tag/5.7.0)

Let's test the changes onto phpMyAdmin before it is released by the end of the month
